yt_dlp/extractor/twitcasting.py

```diff
@@ -89,9 +91,25 @@ def _real_extract(self, url):
             video_js_data = video_js_data[0]
             m3u8_url = try_get(video_js_data, lambda x: x['source']['url'])
 
+        stream_server_data = self._download_json(
+            'https://twitcasting.tv/streamserver.php?target=%s&mode=client' % uploader_id, video_id,
+            'Downloading live info', fatal=False)
+
         is_live = 'data-status="online"' in webpage
+        formats = []
         if is_live and not m3u8_url:
             m3u8_url = 'https://twitcasting.tv/%s/metastream.m3u8' % uploader_id
+        if is_live and has_websockets and stream_server_data:
+            qq = qualities(['base', 'mobilesource', 'main'])
+            for mode, ws_url in stream_server_data['llfmp4']['streams'].items():
+                formats.append({
+                    'url': ws_url,
+                    'format_id': 'ws-%s' % mode,
+                    'ext': 'mp4',
+                    'preference': -100,
```

Comment:
why? Isnt the whole point that this format is better than the m3u8 one?

Reply:
Oh, you're right. It should be prioritized over HLS ones.

Reply:
Since this doesn't have codec or bitrate data available, it will naturally be sorted lower than the other formats. If you know that websocket formats are always better than m3u8 ones, consider increasing its `quality` (not `priority`)